libs/sdk-backend-tiger/src/auth.ts

```diff
@@ -312,9 +316,17 @@ export function createTigerAuthenticationUrl(
         returnAddress = location.href;
     }
 
-    return `${host}${authenticationFlow.loginUrl}?${
+    const baseUrl = `${host}${authenticationFlow.loginUrl}?${
         authenticationFlow.returnRedirectParam
     }=${encodeURIComponent(returnAddress)}`;
+
```

Comment:
this is probably a nitpick, but what about simplifying like this (not tested, just a draft)
```
const params = {
    [authenticationFlow.returnRedirectParam]: returnAddress,
    ...(additionalParams ?? {})
};

const paramsString = Object.entries(params)
    .map(([key, value]) => `${key}=${encodeURIComponent(value)}`)
    .join("&")

return return `${baseUrl}?${params}`
```